docs/api/net.md

```diff
@@ -63,6 +63,28 @@ Creates a [`ClientRequest`](./client-request.md) instance using the provided
 The `net.request` method would be used to issue both secure and insecure HTTP
 requests according to the specified protocol scheme in the `options` object.
 
+### `net.fetch(input[, init])`
+
+* `input` string | [Request](https://nodejs.org/api/globals.html#request)
+* `init` [RequestInit](https://developer.mozilla.org/en-US/docs/Web/API/fetch#options) (optional)
+
+Returns `Promise<GlobalResponse>`.
+
+Sends a request, similarly to how `fetch()` works in the renderer, using
+Chrome's network stack. This differs from Node's `fetch()`, which uses an
+entirely separate HTTP stack, with different limitations.
```

Comment:
I think it'd be useful to include a short explainer on why this might be useful. I have a few ideas, but I imagine many readers would not be sure when to use this.

Reply:
That's a good idea. I'm not sure exactly what I'd suggest though, other than "this is a nicer API than net.request" :P

My main impetus for adding this is because it fits conveniently into later plans I have for `protocol.handle`. What do you think some good things to suggest here would be?

Reply:
Using Chrome's network stack can be better for handling network proxies, right? Maybe something about that. Also, would it allow requests made using this API to be intercepted with [webRequest](https://www.electronjs.org/docs/latest/api/web-request)?

Reply:
Ah, you mean to contrast it with Node's net stack, rather than to contrast it with `net.request`. That makes sense, though since `net.fetch` and `net.request` use the same network stack, I think such a distinction belongs at the top of net.md, as opposed to here in the `fetch` documentation (though we should link it here).

Since that would be additional documentation for both `net.request` and `net.fetch`, I'd like to do that in a followup PR.